rules/do-you-know-how-to-use-single-codebase-for-multiple-domains-with-tinacm-and-nextjs/rule.md

```diff
@@ -0,0 +1,163 @@
+---
+type: rule
+tips: ""
+title: Do you know how to use Single Codebase for Multiple Domains with TinaCMS
+  and Next.js
+seoDescription: Optimize your Next.js and TinaCMS project with a single codebase
+  to efficiently manage multiple domains, locations, and content.
+uri: do-you-know-how-to-use-single-codebase-for-multiple-domains-with-tinacm-and-nextjs
+authors:
+  - title: ""
+created: 2024-08-28T17:37:00.000Z
+guid: 2bba887b-9dd7-4f07-978c-a793611e2f85
+---
+## Introduction
```

Comment:
@PothieuG 

SSW Rules start with a intro paragraph, no need for heading.

- [ ] Please remove heading "Introduction"